Jenkinsfile

```diff
@@ -1,5 +1,8 @@
-buildPlugin(
-  useContainerAgent: true,
+/*
+ See the documentation for more options:
+ https://github.com/jenkins-infra/pipeline-library/
+*/ buildPlugin(
+  useContainerAgent: true, // Set to `false` if you need to use Docker for containerized tests
```

Comment:
Useless.
```suggestion
buildPlugin(
  useContainerAgent: true,
```

Reply:
Archetype must be adapted then : https://github.com/jenkinsci/archetypes/blob/master/common-files/Jenkinsfile#L1a

https://github.com/openrewrite/rewrite-jenkins/blob/main/src/main/resources/META-INF/rewrite/rewrite.yml#L62

At some point we just need to decide to keep such comment or not.

Reply:
* the rewrite is missing `forkCount`
* would be nicer if the rewrite did not touch a file which was materially the same (ignoring whitespace)
* the proposed rewrite had mangled whitespace after the comment

Reply:
FWIW I think the comment is fine in the _archetype_, because that is deliberately intended as a skeleton for a brand-new plugin showing you some things you might consider doing and explaining what they mean if you have never encountered them before. Seems less helpful in the OpenRewrite proposed text. Would be less intrusive to have a single-line comment like

```groovy
// Reference: https://github.com/jenkins-infra/pipeline-library/blob/f26089381c825325666e3f2f1b31e35e9f63c2c7/README.adoc#buildplugin
```

(Here I am using the permalink since the docs might well be rearranged in the future. Or just the repo link would suffice I guess.)